appinfo/routes.php

```diff
@@ -56,5 +56,8 @@
 		['name' => 'settings#setConfig', 'url' => '/v1/config/{key}', 'verb' => 'POST'],
 		// Tools
 		['name' => 'email#sendEmailPublicLink', 'url' => '/v1/public/sendmail', 'verb' => 'POST'],
+	],
+	'resources' => [
+		'appointmentConfig' => ['url' => 'api/appointment_configs']
```

Comment:
No `/v1/` prefix?

Reply:
It's an internal API. What's the purpose of the versioning? if the API changes we won't keep two versions anyway, right?